### PR TITLE
[Create Folder] Elements

### LINKED ISSUE #178: [Create Folder] Elements
Add endpoint to create folder for different element types
Part of https://github.com/pimcore/studio-ui-bundle/issues/71

### PR DESCRIPTION
**Pull request by lukmzig**
## Changes in this pull request
Resolves https://github.com/pimcore/studio-backend-bundle/issues/178

### CODE REVIEW

File: src/Element/Controller/FolderController.php [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    #[DefaultResponses([
+        HttpResponseCodes::UNAUTHORIZED,
+        HttpResponseCodes::NOT_FOUND,
+    ])]
+    public function createFolder(
+        int $parentId,
+        string $elementType,
+        #[MapRequestPayload] FolderData $folderData
+    ): Response {
+        $user = $this->securityService->getCurrentUser();
```
Comment (mattamon) [inline src/Element/Controller/FolderController.php]: User is kinda redundant here? :)

**Review by mattamon — CHANGES_REQUESTED** [review]
Rest LGTM :)

File: src/Element/Controller/FolderController.php (hunk 1 above)
Comment (lukmzig) [reply src/Element/Controller/FolderController.php]: true that :)

**Review by mattamon — APPROVED** [review]
🦭

**Pull request merged**